src/components/elements/side-tab.js

```diff
@@ -35,29 +35,29 @@ const TabList = styled.ol`
         width: 100%;
     }
 `
-
 const TabContent = styled.div`
     flex: 1;
 `
 
 const StyledTab = styled.li`
     cursor: pointer;
-    padding: 1.8rem 0;
-    border-bottom: 1px solid var(--color-red-2);
+    padding: ${props => props.padding || '0 0 2.4rem 0'};
+    width: ${props => props.item_width || '38rem'};
 
     & > p {
-        color: var(--color-red-2);
+        color: var(--color- ${props => props.font_color || 'black-3'});
+        opacity: 0.32;
+        font-size: ${props => props.font_size || 'var(--text-size-m)'};
+        padding-left: 1.6rem;
+        max-width: 33rem;
     }
-    &.tab-active {
-        border-bottom: 1px solid var(--color-red);
-
-        & > p {
-            color: var(--color-red);
-        }
+    &.tab-active > p {
+        opacity: 1;
+        color: var(--color- ${props => props.active_font_color || 'black'});
```

Comment:
this will not work actually, bcause there is a space after `--color-`, can use prettier ignore for this